.github/workflows/reusable-e2e.yml

```diff
@@ -327,7 +334,7 @@ jobs:
 
   e2e-ios:
     needs: check-changed-packages
-    if: ${{ needs.check-changed-packages.outputs.has-changed-packages == 'true' }}
+    if: ${{ inputs.skip-changed-packages-check || needs.check-changed-packages.outputs.has-changed-packages == 'true' }}
```

Comment:
why don't we just remove the condition if we "always run e2e tests for native platform on main"?

Reply:
Oh, NVM, I forgot we use the the re-usable-e2e at other places

Reply:
We still want to skip on PRs but leaving it running async on main so that we can catch ❌ in case some changes seems unrelated but in reality they are related